src/ucp/core/ucp_ep.h

```diff
@@ -393,31 +393,40 @@ typedef struct {
 } ucp_ep_ids_t;
 
 
-/*
- * Endpoint extension for generic non fast-path data
+/**
+ * Endpoint extension for control data path
  */
 typedef struct {
-    ucp_ep_ids_t                  *ids;          /* Local and remote IDS, TODO:
-                                                    remove indirect pointer after
-                                                    stride allocator improvement */
-    void                          *user_data;    /* User data associated with ep */
-    ucs_list_link_t               ep_list;       /* List entry in worker's all eps list */
+    ucp_ep_ids_t                  ids;           /* Local and remote IDs */
     ucp_err_handler_cb_t          err_cb;        /* Error handler */
 
-    /* Endpoint match context and remote completion status are mutually exclusive,
-     * since remote completions are counted only after the endpoint is already
-     * matched to a remote peer.
+    /* Endpoint match context is mutually exclusive with:
```

Comment:
is this comment relevant now? match is not here and they are not in the union anymore

Reply:
yes, should remove it
and use the old one